modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -619,6 +633,35 @@ private void _loadObjectDefinition(Map<String, Serializable> parameters)
 		throw new NotFoundException("Missing parameter \"objectDefinitionId\"");
 	}
 
+	private void _validateObjectProperties(
+		ObjectEntry objectEntry, ObjectDefinition objectDefinition,
+		Locale locale) {
+
+		Stream<String> stream = objectEntry.getProperties(
+		).keySet(
+		).stream();
+
+		List<String> propertieNames = stream.collect(
+			Collectors.toCollection(ArrayList::new));
```

Comment:
No need to cast ArrayList, should be Collectors.toList()

Reply:
Thanks for your review. I'll fix this.